wp-content/themes/ipbs-largo/partials/member-map.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Displays the member map, and enqueues the relevant assets.
+ */
+
+$path = get_stylesheet_directory() . '/images/map.svg';
+
+wp_register_style(
+	'ipbs-map-css',
+	get_stylesheet_directory_uri() . '/css/map.css',
+	null,
+	filemtime( get_stylesheet_directory() . '/css/map.css' )
+);
+wp_register_script(
+	'ipbs-map-animate-dependency',
+	'https://cdnjs.cloudflare.com/ajax/libs/velocity/1.2.3/velocity.js',
+	null,
+	false,
+	true
+);
+wp_register_script(
+	'ipbs-map-animate',
+	get_stylesheet_directory_uri() . '/js/animate.js',
+	array(
+		'ipbs-map-animate-dependecy',
```

Comment:
> Why isn't any of the JS enqueued, when the CSS is?

There's a missing `n` in `dependency` here

Reply:
Once this is fixed, you will see this error when trying to click on a city:

`animate.js?ver=1580138704:10 Uncaught ReferenceError: Velocity is not defined`

That looks like it's happening because `Velocity` is trying to use `$` for jQuery but since WP defines jQuery as `jQuery` and not `$`, it's not working. We'll probably need to set `var $ = jQuery` somewhere in the animate script.

Reply:
After defining $ = jQuery in https://github.com/INN/umbrella-ipbs/pull/67/commits/386f8066ec7c488ce0126b7f9a5c56dfe9bbb424, the error persists. 

But it looks like `$.Velocity` and `jQuery.Velocity` are both defined, so I'll try that.

Reply:
Fixed in b790ead